ztp/gitops-subscriptions/argocd/Upgrade.md

```diff
@@ -1,43 +1,127 @@
-## Steps to Upgrade from v4.9 [argocd/example](https://github.com/openshift-kni/cnf-features-deploy/tree/release-4.9/ztp/gitops-subscriptions/argocd/resource-hook-example) to v4.10 [argocd/example](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example)
+## Upgrading GitOps ZTP
+The Gitops ZTP infrastructure can be upgraded independently from the underlying cluster, ACM, and Openshift version running on the spoke clusters. This procedure guides the upgrade process to avoid impact on the spoke clusters, however any changes to the content or settings of policies, including adding recommended content will result in changes that must be rolled out and reconciled to the spokes.
 
-Follow the steps 1, 2 and 3 to prepare the hub cluster as [Readme](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md) (Preparation of Hub cluster) section mentioned.
- 
-**Upgrading the SiteConfig repository:**
-In the siteConfig repository follow the steps below:
- 1. Delete the post-sync.yaml and pre-sync.yaml files.
- 1. If the siteconfig*.yaml in the repository contain definition for the Namespace remove it.
- 1. Update the kustomization.yaml file to include all SiteConfig yaml files in the `generators` section as below
+At a high level the strategy for upgrading the GitOps ZTP infrastructure is:
+1. Stop the ArgoCD applications
+1. Install the new tooling
+1. Update required content and optional changes in the GIT repository
+1. Update and restart the application configuration
+
+This procedure assumes you have a fully operational Hub cluster running the prior version of the GitOps ZTP infrastructure. If you need to install a new instance of the ZTP infrastructure, follow the instructions in the [Readme](README.md) section (Installing the GitOps Zero Touch Provisioning pipeline).
+
+### Upgrade Preparation
+Obtain the new version of the GitOps ZTP container. This container can be custom built using the ../../ztp/resource-generator build directory. Extract the argocd/deployment directory as described in [Preparation of ZTP GIT repository](#preparation-of-ztp-git-repository).
```

Comment:
Better to explain that the user can get the ztp-site-generator image without re-build just by referring to the [Preparation of ZTP GIT repository](#preparation-of-ztp-git-repository) using $ podman pull ztp-site-generator:v4.10 
Also not sure if the url is working ? I believe u need the full url

Reply:
Good point. I'll add a section to the README on how to get the container (pulling from source is the primary but I think it is good to point the user to the README under resource-generator in case they want to build from source) and point to that.
Thanks for the catch on this URL. It references a section in the README so I've updated to correctly point to it.